extension.ts

```diff
@@ -11,38 +11,83 @@ import { showCmdLine } from './src/cmd_line/main';
 import { ModeHandler } from './src/mode/modeHandler';
 import { TaskQueue } from './src/taskQueue';
 import { Position } from './src/motion/position';
+<<<<<<< aa8f6051a847c442e374ce22920ca5bd21835aec
+=======
+
+export class EditorIdentity {
+    private _fileName: string;
```

Comment:
Isn't it sufficient to ALWAYS compare filename AND column number? When don't we need to?